### PR TITLE
Mention Portals + Testing Library workaround on README 💬

### PR DESCRIPTION
**Pull request by lluia**
Following [this suggestion](https://github.com/nickcolley/jest-axe/issues/105#issuecomment-650165702) 💯

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,17 @@ it('should demonstrate this matcher`s usage with react testing library', async (
 > Note: If you're using `react testing library` <9.0.0 you should be using the
 > [`cleanup`](https://testing-library.com/docs/react-testing-library/api#cleanup) method. This method removes the rendered application from the DOM and ensures a clean HTML Document for further testing.
 
+In case you're using this library to verify some component that relies on [React Portals](https://reactjs.org/docs/portals.html#gatsby-focus-wrapper), make sure you supply [`baseElement`](https://testing-library.com/docs/react-testing-library/api#baseelement) instead of `container`:
```
Comment (nickcolley) [inline README.md:108]:
```suggestion
If you're using [React Portals](https://reactjs.org/docs/portals.html), use the [`baseElement`](https://testing-library.com/docs/react-testing-library/api#baseelement) instead of `container`:

```

File: README.md [hunk 2]
```diff
@@ -105,6 +105,17 @@ it('should demonstrate this matcher`s usage with react testing library', async (
...
+```js
+it('should work with React Portals as well', async () => {
+  const { baseElement } = render(<App/>)
+  const results = await axe(baseElement)
+  
+  expect(results).toHaveNoViolations()
+})
+```
+Otherwise, `jest-axe` won't be able to perform its clean-up properly and throw an error.
```
Comment (nickcolley) [inline README.md:117]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -105,6 +105,17 @@ it('should demonstrate this matcher`s usage with react testing library', async (
 > Note: If you're using `react testing library` <9.0.0 you should be using the
 > [`cleanup`](https://testing-library.com/docs/react-testing-library/api#cleanup) method. This method removes the rendered application from the DOM and ensures a clean HTML Document for further testing.
 
+In case you're using this library to verify some component that relies on [React Portals](https://reactjs.org/docs/portals.html#gatsby-focus-wrapper), make sure you supply [`baseElement`](https://testing-library.com/docs/react-testing-library/api#baseelement) instead of `container`:
+```js
+it('should work with React Portals as well', async () => {
+  const { baseElement } = render(<App/>)
```
Comment (nickcolley) [inline README.md:111]: I'm wondering if we could do this for all examples?

Is there any downside to always using this baseElement?

**Review by nickcolley — APPROVED** [review]
Nice!

I've added some suggestions to simplify the wording.

Let me know what you think about adopting 'baseElement' for all examples...

File: README.md (hunk 2 above)
Comment (nickcolley) [reply README.md:117]: I think we can leave this out and focus on just mentioning portals.

File: README.md (hunk 3 above)
Comment (nickcolley) [reply README.md:111]: If this did work, we dont need to mention portals at all, just update the existing examples.

Comment (lluia) [reply README.md:112]: I see these days on the [`testing-library` docs](https://testing-library.com/docs/react-testing-library/example-intro) they use `screen` on their examples most of the time instead of `container`. 

Maybe we could use that instead of `baseElement` on the examples, I need to double check though it doesn't break the tests with portals though 👍🏽

**Comment by nickcolley** [comment]
Thanks again!
